meson.build

```diff
@@ -74,18 +75,33 @@ if get_option('wayland') != 'false'
   endif
 endif
 
+# WIN32
+use_win32 = false
+if get_option('win32') != 'false'
+  require_win32 = get_option('win32') == 'true'
+  use_win32 = dependency('libva-win32', required: require_win32).found()
+  libva_utils_flags = [ '-DHAVE_VA_WIN32=1', '/std:c++20' ]
```

Comment:
I always thought that `/` options syntax is something very much windows specific. Will '/std:c++20' actually work with mingw?

Also, is there any standard way to specify required c++ standard in meson?

Reply:
It does build on both MSVC and MINGW. The way to specify std in meson is in libva-utils/meson.build in the project() tag, adding 'cpp_std=c++XX'. The original project is targetting C++11 in linux (test/meson.build has -std=c++11) so I just went ahead and replaced the only small C++20 usage with C++11 compatible code and was able to remove this altogether.